docs/benchmarks.md

```diff
@@ -0,0 +1,125 @@
+# Benchmarks
```

Comment:
should we have a link to this from integration test README and vice versa if it needs the setup stuff from that?